02_bubble/joel00wood/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+)
+
+func bubble(s []int) []int {
+	c := make([]int, len(s))
```

Comment:
This is fine, but you can also copy slice using onliner: `c := append(s[:0:0], s...)`. No action required.

Reply:
I'll keep that in mind in future